tools/eslint-rules/require-common-first.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @fileoverview Require `common` module first in our tests.
+ */
+'use strict';
+
+const path = require('path');
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+  const requiredModule = 'common';
+  const isESM = context.parserOptions.sourceType === 'module';
+  const foundModules = [];
+
+  /**
+   * Function to check if a node is a string literal.
+   * @param {ASTNode} node The node to check.
+   * @returns {boolean} If the node is a string literal.
+   */
+  function isString(node) {
+    return node && node.type === 'Literal' && typeof node.value === 'string';
+  }
+
+  /**
+   * Function to check if a node is a require call.
+   * @param {ASTNode} node The node to check.
+   * @returns {boolean} If the node is a require call.
+   */
+  function isRequireCall(node) {
+    return node.callee.type === 'Identifier' && node.callee.name === 'require';
+  }
+
+  /**
+   * Function to check if the path is a required module and return its name.
+   * @param {String} str The path to check
+   * @returns {String} required module name
+   */
+  function getRequiredModuleName(str) {
+    if (str === '../common/index.mjs') {
+      return 'common';
+    }
+
+    return path.basename(str);
+  }
+
+  /**
+   * Function to check if a node has an argument that is a required module and
```

Comment:
I think both this description and the function name could be improved by removing "required". Same for the function above.

Reply:
Fixed. Thanks~